### DISCUSSION TITLE
Colors for passive LCD widgets

### DISCUSSION THREAD

**Discussion by Ratfink** [hydrogen-music/hydrogen#1192] [Ideas]
I've recently been working on making more LCD widgets interactive in #1176, with that branch currently enabling the scroll wheel on many of the LCD spin boxes in the instrument editor.  However, as @theGreatWhiteShark pointed out, it was initially unclear which LCDs are interactive, and which are just passive displays.  To address this, I'm considering several different color schemes to give the non-interactive LCDs a "backlight off" look.  Please take a look at the options below and tell us what you think!

### Current colors (for reference)
![Screenshot from 2021-02-23 18-48-40](https://user-images.githubusercontent.com/320472/108923984-ccef1000-7607-11eb-8af6-9c6a78624a09.png)

### Palette 1
![Screenshot from 2021-02-18 21-14-56](https://user-images.githubusercontent.com/320472/108922612-5fda7b00-7605-11eb-819c-34ddeecc022f.png)

### Palette 2
![Screenshot from 2021-02-23 18-24-02](https://user-images.githubusercontent.com/320472/108922615-60731180-7605-11eb-8a9c-08b141d7df60.png)

### Palette 3
![Screenshot from 2021-02-23 18-42-21](https://user-images.githubusercontent.com/320472/108923490-e93e7d00-7606-11eb-9080-5d474531e47d.png)

**Comment by oddtime** [comment]
Hi, I prefer palette 3. What about a clearer background to enhance contrast with the letters?

**Comment by theGreatWhiteShark** [comment]
I like palette 3 most too.

How about swapping foreground and background color to have high contrast while making the background darker?

![lcd_color_set2](https://user-images.githubusercontent.com/14164547/108999730-01fc7080-76a3-11eb-9392-836b61b7a46e.png)

or 

![lcd_color_set3](https://user-images.githubusercontent.com/14164547/108999745-06c12480-76a3-11eb-92d8-8be7746915c2.png)

**Reply by trebmuh** [reply]
I very much like the 2nd!

**Reply by duck57** [reply]
Neither option is bad but I agree that option 2 gives a better "off" look compared to 1 where it says "no plugin"
